projects/plugins/boost/app/assets/src/js/features/cornerstone-pages/meta/meta.tsx

```diff
@@ -215,11 +215,21 @@ const List: React.FC< ListProps > = ( {
 		recordBoostEvent( 'cornerstone_pages_save', {
 			list_length: inputValue.split( '\n' ).length,
 		} );
+		setNotice( {
+			id: 'cornerstone-pages-save',
+			type: 'success',
+			message: __( 'Cornerstone pages saved', 'jetpack-boost' ),
+		} );
```

Comment:
This should be a mutation notice and only show as a success after a confirmation about it being saved is received.

Reply:
Thanks, fixed that by moving it into the right callback.